src/routes/PaymentResponsePage.tsx

```diff
@@ -175,11 +177,12 @@ export default function PaymentResponsePage() {
                 {t("errorButton.close")}
               </Button>
             </Box>
-            {outcome === ViewOutcomeEnum.SUCCESS && (
-              <Box sx={{ width: "100%" }} px={{ xs: 8, sm: 0 }}>
-                <SurveyLink />
-              </Box>
-            )}
+            {conf.CHECKOUT_SURVEY_SHOW === "true" &&
+              outcome === ViewOutcomeEnum.SUCCESS && (
```

Comment:
Can we avoid to use the string "true" or "false" if it's not possible to use a boolean type? Something like "show" or "hidden"